sycl/source/detail/queue_impl.hpp

```diff
@@ -317,6 +311,13 @@ class queue_impl {
             ext::oneapi::cuda::property::queue::use_default_stream>()) {
       CreationFlags |= __SYCL_PI_CUDA_USE_DEFAULT_STREAM;
     }
+    if (getPlugin().getBackend() == backend::ext_oneapi_level_zero &&
+        MPropList
+            .has_property<ext::oneapi::property::queue::discard_events>()) {
+      // Pass this flag to the Level Zero plugin to be able to check it from
```

Comment:
Why is this L0 specific? Why should we not pass it to the other plugins? (I think we should)

Reply:
Fixed to pass this property to all backends.